scripts/ci/build-and-test.sh

```diff
@@ -10,6 +10,26 @@ cd $(dirname $0)/../..
 source scripts/ci/sources/mode.sh
 source scripts/ci/sources/tunnel.sh
 
+# Get commit diff
+if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+  diff=$(git diff --name-only $TRAVIS_COMMIT_RANGE)
+else
+  diff=$(git diff --name-only $TRAVIS_BRANCH...HEAD)
+fi
+
+# Check if tests can be skipped
+skip_tests=true
+for filename in $diff; do
+  if ! [[ "$filename" =~ .*\.md ]]; then
+    skip_tests=false
+    break
+  fi
+done
```

Comment:
I am confused... ``+``  here is used to tackle the empty commit case (e.g. trigger CI with an empty commit)... 
If it's an empty commit, ``*`` would skip the tests, but ``+`` would't. The feature here is to skip tests only ``md`` files are changed. That's why I use ``+``.